marshmallow/utils.py

```diff
@@ -334,16 +333,34 @@ def callable_or_raise(obj):
     return obj
 
 
-def get_func_args(func):
-    """Given a callable, return a tuple of argument names. Handles
+def get_func_args2(func):
+    """Given a callable, return a list of argument names. Handles
```

Comment:
Okay I think I misunderstood about the rebase. I did rebase but not with that branch, but with marshmallow/dev. I will try again. Do you have a tutorial at hand on how rebases easily?